File: src/engsoft/jogo/patos/Nao_Voa.java
package engsoft.jogo.patos;

public class Nao_Voa implements Padrao_Voaveis { // Classe concreta que implementa a interface Padrao_Voaveis

	
	public Nao_Voa() // Código em tempo de implementação, inicializando o objeto
	{
			
	}
	
	public String voar() {
		return "Esse pato não Voa. Velocidade: " + getVelocidade();
	}

	public double getVelocidade() {
		return 0;
	}

}

File: Nao_Voa.ts
import { Padrao_Voaveis } from "./Padrao_Voaveis";

export class Nao_Voa implements Padrao_Voaveis {
    voar(): string {
        return "Este pato não voa.";
    }

    getVelocidade(): number {
        return 0;
    }
}
